packages/react/src/types.ts

```diff
@@ -83,4 +85,6 @@ export interface SignUpButtonProps extends ButtonProps {
 
 export type SignInWithMetamaskButtonProps = Pick<ButtonProps, 'redirectUrl' | 'children'>;
 
-export type RedirectToProps = RedirectOptions;
+export type RedirectToSignInProps = SignInRedirectOptions;
+export type RedirectToSignUpProps = SignInRedirectOptions;
+export type RedirectToProps = RedirectToSignInProps | RedirectToSignUpProps;
```

Comment:
❓ Do we still need this type ?

Reply:
It was exported so it's breaking to remove it, no?

Reply:
Inside index.ts only a few types are exported from `types` and this is not one of them, seems to be it is safe to remove if the internals do not require it.

Reply:
I'm removing it then :D. Thanks